### PR TITLE
Spiker bos -> lua

### PR DESCRIPTION
**Pull request by Arch-Shaman**
Did this for future wars, so I'm pring to base in case you want it.

WIP:
- [x] Testing
- [ ] RestoreAfterDelay?

### CODE REVIEW

**Comment by Arch-Shaman** [comment]
Tested via future wars: https://cdn.discordapp.com/attachments/440166152253997056/1029159948938133575/2022-10-10_18-33-28.mp4

Can PR the sound too if you would like it.

File: scripts/chickens.lua [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+	Turn(lthigh, x_axis, 0, math.rad(200))
+	Turn(lshin, x_axis, 0, math.rad(200))
+	Turn(rshin, x_axis, 0, math.rad(200))
+	Turn(lknee, x_axis, 0, math.rad(200))
+	Turn(rknee, x_axis, 0, math.rad(200))
+end
+
+local function RecoilThread()
+	Turn(lsack, y_axis, math.rad(40), math.rad(440)) -- this had to be changed because it looked bad
+	Turn(rsack, y_axis, math.rad(-40), math.rad(440)) -- ditto. 1 isn't fast enough!
```
Comment (sprunk) [inline scripts/chickens.lua:101]: Stick to the original values?
```suggestion
	Turn(lsack, y_axis, math.rad(40), math.rad(360))
	Turn(rsack, y_axis, math.rad(-40), math.rad(360))
```

File: scripts/chickens.lua [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+		Turn(rthigh, x_axis, math.rad(-20), math.rad(210))
+		Turn(rknee, x_axis, math.rad(-60), math.rad(210))
+		Turn(rshin, x_axis, math.rad(50), math.rad(210))
+		Turn(rfoot, x_axis, math.rad(30), math.rad(210))
+
+		Turn(body, z_axis, math.rad(5), math.rad(20))
+		Turn(lthigh, z_axis, math.rad(-5), math.rad(20))
+		Turn(rthigh, z_axis, math.rad(-5), math.rad(20))
+		Move(body, y_axis, 0.7, 4000)
```
Comment (sprunk) [inline scripts/chickens.lua:30]: Stick to the original values (also avoid trailing whitespace):
```suggestion
		Move(body, y_axis, 0.7, 11)
```

**Comment by sprunk** [comment]
Changes from the review done in https://github.com/ZeroK-RTS/Zero-K/commit/8fa91cb73f200b76b7735f08a8068571e4405e84

**Pull request merged**